modules/gdscript/gdscript_editor.cpp

```diff
@@ -2172,7 +2172,7 @@ static void _find_identifiers(const GDScriptCompletionContext &p_context, bool p
 	}
 
 	static const char *_type_names[Variant::VARIANT_MAX] = {
-		"null", "bool", "int", "float", "String", "Vector2", "Rect2", "Vector3", "Transform2D", "Plane", "Quat", "AABB", "Basis", "Transform",
```

Comment:
I think the order should be the same as in the `Variant::TYPE` enum. I see that `"StringName"` is missing too, could you add it?